src/components/shared/mediaText.js

```diff
@@ -203,6 +204,7 @@ function MediaText (props) {
     }
     headingClass = classNames(headingClass, headingColor);
     textCol = classNames(textCol, textColBg, textColHeight, textColPadding, "text-break");
+    if (HeadingLevel === "h5" || HeadingLevel === "h6") HeadingLevel = "h4";
 
```

Comment:
Do we want to restrict users from using levels below h4? This line from the help text:

> We strongly advise you refrain from choosing a level below h4 so levels h5 and h6 are preserved for Accordion Item titles and their subsections.

...was originally meant for Section Titles. We didn't want people using anything below h4 for Section Titles because then there would be inadequate levels remaining for Accordions within those Sections. Media and Text widgets, however, won't have that issue since you can't insert an Accordion within them.